build.gradle

```diff
@@ -0,0 +1,252 @@
+import org.gradle.internal.os.OperatingSystem;
+import groovy.io.FileType
+
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java project to get you started.
+ * For more details take a look at the Java Quickstart chapter in the Gradle
+ * user guide available at https://docs.gradle.org/4.9/userguide/tutorial_java_projects.html
+ */
+buildscript {
+    dependencies {
+        classpath group: 'de.dynamicfiles.projects.gradle.plugins', name: 'javafx-gradle-plugin', version: '8.8.2'
+
+    }
+    repositories {
+        mavenLocal()
```

Comment:
Should we delete these? Pretty sure we don't use maven at all

Reply:
nvm it breaks if I delete it, junit is fetched from maven apparently. Maven is generally better than what we've done in the past, which is copying jar files from the internet

Reply:
Yeah, good idea. Also the auto generated comments. I'll try it. I don't know if the plugins are pulled from there...